samples/interfaces.go

```diff
@@ -0,0 +1,44 @@
+// Interfaces
+package main
+
+import "fmt"
+
+type scaler interface {
+	scale(float64)
+}
+
+func main() {
+	var s scaler
+	s = &rect{2, 3}
```

Comment:
speaker notes: ask the class, "why didn't we do `s := &rect{2, 3}` instead of these two lines?"

Reply:
Great idea!
Will do.